### PR TITLE
CleanUpGoReturn

### PR DESCRIPTION
**Pull request by satyamamrev**

### CODE REVIEW

**Comment by AbdealiJK** [comment]
@sils1297 @satyamamrev needs to be added to the org and assigned to https://github.com/coala-analyzer/coala/issues/1516

**Comment by Redridge** [comment]
The easiest way to fix all of the problems would be to run coala with SpaceConsistencyBear and PEP8Bear like so

```
$ coala --files=src/test.py --bears=SpaceConsistencyBear,PEP8Bear
```

**Comment by sils1297** [comment]
in this case just running `coala` will suffice, it'll auto pick up the config.

**Comment by sils1297** [comment]
@satyamamrev hi, I've sent you an invitation to the org, sorry that took so long I had an exam today. You know how to improve your PR? There has been plenty of review comments, your commits need to be squashed and this needs a rebase. If you need help doing that (most people do :)) just come by at https://gitter.im/coala-analyzer/coala , there's probably somebody around who can help you!

**Comment by sils1297** [comment]
that said welcome! : )

File: bears/tests/go/GoReturnsBearTest.py [hunk 1]
```diff
@@ -28,13 +28,9 @@
...
-     'func main() {', '\tf, _ := os.Open("foo")', '}']
+
 
 GoReturnsBearTest = verify_local_bear(
     GoReturnsBear,
-    (good_file1,
-     good_file2,),
-    (bad_file1, bad_file2,))
+    (good_file1,good_file2,),
+    (bad_file1, bad_file2,))
```
Comment (sudheesh001) [inline bears/tests/go/GoReturnsBearTest.py]: Leave a new line here after the  `))`

Comment (satyamamrev) [inline bears/tests/go/GoReturnsBearTest.py]: done

**Comment by sils1297** [comment]
hey, you sure you don't want some help? Here's the link again https://gitter.im/coala-analyzer/coala

**Comment by sils1297** [comment]
essentially we'd want you to fetch from upstream, do an interactive rebase to squash all your commits, make sure coala runs without errors and push force it up.

**Comment by sils1297** [comment]
Hi, @hypothesist is now working on this issue. If you want to work on an issue, please accept the invitation to our org so we can assign you. We don't have the ability to manage issues/prs from people that aren't assigned because everything is freakinly active here nowadays.

**Pull request closed without merging**